Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:latest AS golang
+WORKDIR /go/src/github.com/bitly/oauth2_proxy
+RUN go get -u github.com/golang/dep/cmd/dep
+COPY . .
+#RUN dep ensure && \
```

Comment:
If this repo isn't compatible with `dep`, this needs to happen soon imo. I'll not block this PR with it though.  Please create an issue on this repo to track.